hudi-common/src/main/java/org/apache/hudi/common/config/DFSPropertiesConfiguration.java

```diff
@@ -174,6 +174,11 @@ public static TypedProperties getGlobalProps() {
     return globalProps;
   }
 
+  public static TypedProperties addToGlobalProps(String key, String value) {
```

Comment:
If this is only needed tests, then maybe define there.

Reply:
`GLOBAL_PROPS` is a private variable so it is kind of hard to define it in the test class. I added a `test only` comments there.

Reply:
Got it. I think we should really find a way to include `@VisibleForTesting` annotation just like im hudi-flink module. Maybe you could take it up later.

Reply:
Yes this is a good point. Flink has its own @VisitbleForTesting. However, in hudi-common package, I think Hudi doesn't want to use Guava as its dependency: https://github.com/apache/hudi/blob/master/style/checkstyle.xml#L270. But for the long term, I agree we should think a way to resolve this issue.